spec/features/registrations_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'feature_helper'
+
+describe "Registrations" do
+  describe 'disable 2FA' do
```

Comment:
Would be nice to have a test where the 2FA is enabled. Do you think that it is easy to add on this PR?